portfolio/src/main/webapp/art.html

```diff
@@ -0,0 +1,114 @@
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+
+
+    <title>Chiamaka Chukwuka</title>
+
+
+    <link href="https://fonts.googleapis.com/css?family=Princess+Sofia" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Miss+Fajardose" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Ewert" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Barcode+39+Extended+Text" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Cinzel+Decorative" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+
+
+  </head>
+  <body>
+
+
+    <div id="content">
+
+
+    <div id="logo"><em><h1 id="first"><div id="altletters"><span id="c1">C</span><span id="c12">hi</span><span id="c2">&#10037;a<span id="c23">&#10037;</span></span><span id="c3">m</span><span id="c4">a</span><span id="c45">&#10037;</span><span id="c5">k</span><span id="c6">a</span></div></h1></em></div>
+    <h3 id="subhead"> &#9830; Chiamaka  (Chee-uh-mok-uh)  Chukwuka  (Chew-kwoo-kah). &#9830;</h3>
+    <script src="script.js"></script>
+
+    
+    </div>
+    <head><title>Artwork</title></head>
+    <body>
+      <link href="https://fonts.googleapis.com/css?family=Playfair+Display+SC" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Homemade+Apple" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Monsieur+La+Doulaise" rel="stylesheet">    
+      <link href="style.css" rel="stylesheet" type="text/css" />
+      <header>
+        <ul class="navigation">
+            <a href="./index.html"><li>HOME</li></a>
+            <a href="./projects.html"><li>PROJECTS</li></a>
+            <a href="./art.html"><li>ARTWORK</li></a>
+            <a href="./aboutme.html"><li>ABOUT ME</li></a>
+        </ul>
+      </header>
+    <div class ="artbody">
+
+        <p> padding</p>
+        <p> padding</p>
+
+
+        <div class="separator" style="clear: both; text-align: center;">
+        <img border="0" height="900" src="./images/feet.jpg"  width="620" />
+        </div>
+        <p id="desc"> "NOSTALGIA"</p> 
+        <p> padding</p>
```

Comment:
You can use \<br\> here.  Is this actually displaying "padding" or did you hide that somehow?

Reply:
Okay I fixed this issue now.

Reply:
resolving because I fixed all the <p>padding</p> with <br>